rstudio-server-docker/docker-compose.yml

```diff
@@ -7,13 +7,13 @@ services:
     ulimits:
       nofile: 10000
     ports:
-        - "8889:8787"
+      - "8889:8787"
     volumes:
       # mount conda env into exactly the same path as on the host system - some paths are hardcoded in the env.
       - /home/centos/.conda/envs/seurat-knitr:/home/centos/.conda/envs/seurat-knitr
```

Comment:
```suggestion
       # TODO: adjust to your environment name
      - $CONDA_PREFIX/envs/seurat-knitr:$CONDA_PREFIX/envs/seurat-knitr
```

Reply:
I would leave it like this, because it allows people to test the default setup

Reply:
It's less error prone to use the environment variable in case somebody has installed `conda` at a different location.

Reply:
fair enough.